app/controllers/tasks_controller.rb

```diff
@@ -4,15 +4,6 @@ class TasksController < ApplicationController
   before_action :set_users, only: [:new, :edit]
   before_action ->{ ensure_ownership(@task) }, only: [:edit, :update, :destroy]
 
-  def index
```

Comment:
While having `index` views for things doesn't make sense in the Unified Dashboard™ view of things, we'll still need to have this index view for the eventual API. So, my suggestion would be to just leave this here and just not link to this from anywhere. That, or we could par this method down to
```ruby
redirect_to :dashboard
```